app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    first_name: str
+    last_name: str
+    id: int

```

Comment:
In a dataclass, fields with default values must come after fields without default values. If `id` is intended to be optional or assigned by the database, consider providing a default value or making it optional.